tests/functional/templates/reading_pod.yaml.j2

```diff
@@ -0,0 +1,18 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: read-test-{{ storage_class }}
+spec:
+  volumes:
+  - name: shared-data
+    persistentVolumeClaim:
+      claimName: pvc-test-{{ storage_class }}
+      readOnly: false
+  containers:
+    - name: read-test-{{ storage_class }}
+      image: public.ecr.aws/ubuntu/ubuntu:latest
```

Comment:
I'd suggest adding a comment if this registry was used because of rate limits of docker hub (or any other registry)